app.css

```diff
@@ -985,4 +985,4 @@ i {
     justify-content: flex-end;
     align-items: center;
   }
-}
\ No newline at end of file
+}
```

Comment:
- [ ] Great job so far. I will leave the comments here please make sure you fix them all. For other reviews please make sure you commit all the code you've been working on so it will be better to point out the code that needs improvement.

### headline section

- [ ] there is a style difference in this section. Please address that.

Your implementation
![image](https://user-images.githubusercontent.com/59416136/154560731-cd6e1e47-8069-47c1-baba-54f51ee4d0d7.png)

Figma's design
![image](https://user-images.githubusercontent.com/59416136/154560810-c00eb04c-9892-47ac-a0c4-61618eff642c.png)

### Projects section

- [ ] there is a style difference in this section. The images have more height than expected which might be causing this. Please address that.

Your implementation
![image](https://user-images.githubusercontent.com/59416136/154561162-f32856e1-b15f-47bf-a61d-9a1630804a79.png)

Figma's design
![image](https://user-images.githubusercontent.com/59416136/154561224-cdef85a3-7f39-4f09-9f98-8d5e33ab992d.png)

Reply:
> * [ ]  Great job so far. I will leave the comments here please make sure you fix them all. For other reviews please make sure you commit all the code you've been working on so it will be better to point out the code that needs improvement.
> 
> ### headline section
> * [ ]  there is a style difference in this section. Please address that.
> 
> Your implementation ![image](https://user-images.githubusercontent.com/59416136/154560731-cd6e1e47-8069-47c1-baba-54f51ee4d0d7.png)
> 
> Figma's design ![image](https://user-images.githubusercontent.com/59416136/154560810-c00eb04c-9892-47ac-a0c4-61618eff642c.png)
> 
> ### Projects section
> * [ ]  there is a style difference in this section. The images have more height than expected which might be causing this. Please address that.
> 
> Your implementation ![image](https://user-images.githubusercontent.com/59416136/154561162-f32856e1-b15f-47bf-a61d-9a1630804a79.png)
> 
> Figma's design ![image](https://user-images.githubusercontent.com/59416136/154561224-cdef85a3-7f39-4f09-9f98-8d5e33ab992d.png)

- [ ] Kindly update the spacing in the headline section just as the previous reviewer requested, there should be spacing between the text and image, kindly check Figma design and  reduce the size of the text aligned to the left
- [ ] The start collaboration button should be aligned to the left 👍🏿